### ISSUE TITLE
data preprocessing and final loss value

### ISSUE THREAD

**Issue by janeyeon** [auspicious3000/SpeechSplit#7]
Hi. I wanted to ask if you performed data normalization of an audio after trimming all the silences!
And if you did, what method did you use?

What was final validation loss of G and P after training is almost done? My result is something like this..
![image](https://user-images.githubusercontent.com/33058149/91376437-006b0080-e858-11ea-8b1d-0b1b10b735e5.png)

**Comment by auspicious3000** [comment]
Please refer to the data preparation code for details

**Comment by steven850** [comment]
@janeyeon how where you able to train both G and P at the same time? willing to share your code?

**Comment by auspicious3000** [comment]
You don't need to train them at the same time.

**Comment by steven850** [comment]
Is there code available to train P, Im only seeing G in the solver.py

**Comment by auspicious3000** [comment]
You can simply replace G with P along with some other minor modifications.

**Comment by c1a1o1** [comment]
Can you make the right demo.pkl file?

**Comment by Buckingham0805** [comment]
Hi . 
Did you do some preprocessing such as trimming silence and cropping  in the training data? Or Have you done some other changes in the code...?
In my expeiment, the final validation loss is much higher than yours. I want to find the reason...
Thank you in advance.

**Comment by auspicious3000** [comment]
All preprocessing steps are in the code, except trimming silence. But I don't think they will make any fundamental difference. Your loss value looks fine.

**Comment by FurkanGozukara** [comment]
@janeyeon @steven850 any ideas how to fix this problem?

thank you

https://github.com/auspicious3000/SpeechSplit/issues/28

**Comment by 9527950** [comment]
> 

Can you share this code of yours? Thank you very much!
